src/common/containers/GameAreaContainer.js

```diff
@@ -216,33 +223,34 @@ export class GameAreaContainer extends Component {
     if (!started) {
       if (this.urlMatchesGameMode('tutorial')) {
         onStartTutorial();
-      } else if (this.urlMatchesGameMode('practice')) {
-        this.tryToStartPracticeGame(match.params.deck);
-      } else if (location.pathname.startsWith('/sandbox')) {
-        onStartSandbox();
+      } else if (firebaseLoaded) {
+        // Wait until we retrieve data from Firebase so we can get
```

Comment:
The body of this if block is too large to be nested, should receive its own function.